freshmaker/config.py

```diff
@@ -427,6 +427,11 @@ class Config(object):
             'desc': 'A list of bundle name-version entries that do not need to have an auto-rebuild '
                     'tag to be rebuilt. This only applies to the HandleBotasAdvisory handler'
         },
+        'flatpak_server_url ': {
+            'type': str,
+            'default': '',
```

Comment:
How do we want to handle it if the string is empty? Is it OK to just ignore the case?

Reply:
Not quite sure about your point. At least there is a default empty string instead of no attribute.

Reply:
I mean, empty string would lead to bad URL to flatpak API - I don't see that handled.

We could skip flatpak handling if this is empty - this could be useful also for some troubleshooting in near future.

Reply:
Ok. Got it. Fixing